src/randomizer_8_16_64.c

```diff
@@ -23,9 +23,7 @@ int randomizer_8_16_64(int **input){
         return 1;
     }
 
-    *input       = malloc(8 * sizeof(int));
-    *(input + 1) = malloc(16 * sizeof(int));
-    *(input + 2) = malloc(64 * sizeof(int));
```

Comment:
Die reservierung von den Array Elementen wird jz wie **input außerhalb passieren. Das Program wies mit unserer letzten Ausführung 9 verlorene Blöcke im Speicher auf, grund dafür war das lokale reservieren vom selben 8_16_64 speicher und das 3 mal -> also zusammen 9 Blöcke.

Lösung für das Problem war hier statt eine lokale reservieung in randomizer eine reservierung außerhalb und eine freigabe nach allen Sorting algorithmen